lib/org/chromium/webidl/ast/Definition.js

```diff
@@ -25,12 +25,17 @@ foam.CLASS({
       of: 'String',
       documentation: 'Array of all sources that composes this definition.',
       name: 'sources',
+      required: true,
+      factory: function() {
+        return [this.ctxSource];
```

Comment:
Since we have `imports: [ 'source? as ctxSource' ]`, I think we should have `this.ctxSource ? [this.ctxSource] : []`